src/olympia/users/templates/users/emails/user_deleted.ltxt

```diff
@@ -1,9 +1,17 @@
-{% load i18n %}{# L10n: This is an email. Whitespace matters! #}{% blocktrans %}Hello,
+{% load i18n %}{% if is_mza_branding %}{# L10n: This is an email. Whitespace matters! #}{% blocktrans %}Hello,
+
+You’re receiving this message because your user account {{ name }} on {{ site_url }} has been deleted. This could have been done automatically if you recently deleted your Mozilla account (we renamed Firefox Accounts to Mozilla accounts).
```

Comment:
The "we renamed" bit might need to be validated by someone. Do we really need that ? It's on extra thing to remove later...

Reply:
It's "strongly recommended" to have messaging about the name transition - the figma link on https://mozilla-hub.atlassian.net/browse/FXA-7993 shows messaging that says "We've renamed Firefox accounts to Mozilla Accounts.  You'll still sign in with the same username and password, and there are no other changes to the products you use. [Learn more](https://support.mozilla.org/kb/firefox-accounts-renamed-mozilla-accounts)".  

I stripped that down because this is an email you get after you've deleted your account so I didn't think telling them about how to log in or our products made much sense.  Maybe we should include the url to find out more though.

Reply:
Ok. I think that's good enough as-is. It's a fairly specific email, no need to be over concerned about it.